File: src/main/java/org/funivan/lologa/algo/ga/genome/GenomeInterface.java
package org.funivan.lologa.algo.ga.genome;

import org.funivan.lologa.algo.find.one.TileFinder;

import java.util.HashMap;

public interface GenomeInterface extends TileFinder {

    GenomeInterface withData(HashMap<String, Double> genome);

    HashMap<String, Double> data();

}

File: src/main/java/org/funivan/lologa/algo/ga/genome/GenomeInterface.kt
package org.funivan.lologa.algo.ga.genome

import org.funivan.lologa.algo.find.one.TileFinder
import java.util.*

interface GenomeInterface : TileFinder {

    fun withData(genome: HashMap<String, Double>): GenomeInterface

    fun data(): HashMap<String, Double>

}
